src/emulator/config.cpp

```diff
@@ -96,6 +96,7 @@ bool serialize(Config &cfg) {
     config_file_emit_single(emitter, "pstv-mode", cfg.pstv_mode);
     config_file_emit_single(emitter, "show-gui", cfg.show_gui);
     config_file_emit_single(emitter, "archive-log", cfg.archive_log);
+    config_file_emit_single(emitter, "size-icon", cfg.size_icon);
```

Comment:
```suggestion
    config_file_emit_single(emitter, "icon-size", cfg.icon_size);
```

Reply:
i have inversed for no have two time same value lol

Reply:
same value where?
in any case, change this and use a slightly different name in the other place (you can just add `_` at the end) but not here

Reply:
i speak for float icon_size, i have set that for no confunded both ^^